extension/src/fileSystem/index.test.ts

```diff
@@ -231,4 +251,18 @@ describe('findOrCreateDvcYamlFile', () => {
       expect.stringContaining(scriptCommand.PYTHON)
     )
   })
+
+  it('should open the dvc.yaml file in the editor', () => {
+    mockedOpenTextDocument.mockResolvedValue({} as TextDocument)
+
+    void findOrCreateDvcYamlFile('/', '/train.py')
```

Comment:
[Q] Don't want to `await` the result of these?

Reply:
This isn't really crucial to starting running the experiments. I don't mind changing it though